pkgs/development/compilers/vlang/default.nix

```diff
@@ -78,7 +78,7 @@ stdenv.mkDerivation {
 
   # vcreate_test.v requires git, so we must remove it when building the tools.
   preInstall = ''
-    mv cmd/tools/vcreate/vcreate_test.v $HOME/vcreate_test.v
+    mv cmd/tools/vcreate/vcreate_init_test.v $HOME/vcreate_init_test.v
```

Comment:
This doesn't seem to be necessary (any more). Either way the vcreate test is skipped.

Reply:
Fair